xarm:
  angle_ref_1: 0.0
  angle_ref_2: 0
  angle_ref_3: -90
  angle_ref_4: 0.0
  angle_ref_5: 0.0
  angle_ref_6: 0.0
  mvvelo: 0.5
  mvacc: 1